### PR TITLE
added n! method into Main.class;

### PR DESCRIPTION
**Pull request by VladiKSorokA**

### CODE REVIEW

File: src/com/company/Main.java [hunk 1]
```diff
@@ -2,7 +2,22 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    public long faktorial (int n){
```
Comment (necrowman) [inline src/com/company/Main.java]: please add comment to each method that you use!

**Review by necrowman — COMMENTED** [review]
some review #2

File: src/com/company/Main.java (hunk 1 above)
Comment (VladiKSorokA) [reply src/com/company/Main.java]: OK

File: src/com/company/Main.java [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 public class Main {
 
+    // this func calculate n!
```
Comment (necrowman) [inline src/com/company/Main.java]: Ok! Accepted!

**Pull request merged**